Gemfile

```diff
@@ -199,7 +197,7 @@ group :red_hat_virtualization, :manageiq_default do
 end
 
 group :qpid_proton, :optional => true do
-  gem "qpid_proton",                    "~>0.30.0",          :require => false
+  gem "qpid_proton",                    "~>0.37.0",          :require => false
```

Comment:
so does this change depend upon https://github.com/ManageIQ/manageiq-rpm_build/pull/326 ?

Reply:
Good question.  We first need to get this in and ensure we have packages built.  Thanks!  Commented there.

Reply:
FYI, epel8 has 0.37.0
https://mirror.steadfastnet.com/epel/8/Everything/x86_64/Packages/q/

I think we'll need to revert this to use it though:
https://github.com/ManageIQ/manageiq-appliance-build/pull/409

If we go el9, we'll need either the one we're building for el9 or the one in epel9, see also:
https://github.com/ManageIQ/manageiq-rpm_build/pull/326#issuecomment-1834094129

Reply:
I have a standalone PR to bump qpid_proton https://github.com/ManageIQ/manageiq/pull/22271 we still need to drop an exclude in appliance_build to not ignore qpid_proton from EPEL

Reply:
cool, I'll merge that and fix the exclude

Reply:
ok, opened https://github.com/ManageIQ/manageiq-appliance-build/pull/559